packages/amplify-category-geo/src/service-utils/resourceParams.ts

```diff
@@ -19,5 +19,5 @@ export enum AccessType {
 
 export enum DataProvider {
     Esri = 'Esri',
-    Here = 'HERE'
```

Comment:
@dayjournal Could you please restore this? The company is called HERE (all caps)

Reply:
Please retain as `HERE`

Reply:
This item has been corrected.